ui/src/components/IndividualsTable.stories.js

```diff
@@ -7,14 +7,22 @@ export default {
   excludeStories: /.*Data$/
 };
 
-const IndividualsTableTemplate = '<individuals-table :fetch-page="queryIndividuals.bind(this)" />';
+const IndividualsTableTemplate = `
+  <individuals-table
+    :fetch-page="queryIndividuals.bind(this)"
+    :delete-item="deleteIndividual.bind(this)"
+  />
+`;
 
 export const Default = () => ({
   components: { IndividualsTable },
   template: IndividualsTableTemplate,
   methods: {
     queryIndividuals(page) {
       return this.query[page - 1];
+    },
+    deleteIndividual() {
```

Comment:
While testing this part, when I click to remove an entry, it doesn't remove anything from the table. Shouldn't it remove the entry from the table?

Reply:
I'm not sure if we should try to replicate everything that the backend does on Storybook, from what I've seen it's usually just used to show how the components look (although they are usually simpler components).

Reply:
My doubt is if to remove en entry is part of the logic of the component, that shouldn't be included on the component itself? Or do we expect to have different behaviours with the delete option depending on where the component is used?

Reply:
Do you mean that the delete function should be inside the component and not passed as a prop?

Reply:
I think that the functionality that removes the entry from the table, not from the server, should be part of it.  When I tested it in storybook I find odd that the entry is still on the table. I'm not sure, though. In any case, if you think it's better this way, we can leave it for now and think about it later.

Reply:
We could remove the entries from the table, but reloading the page after that to fill in the missing entries kind of "overwrites" it. I think if you find it OK and it works on the app, we can discuss it and improve the functionality on Storybook later.

Reply:
Ok! Make sense.